regcomp_internal.h

```diff
@@ -919,7 +918,6 @@ static const scan_data_t zero_scan_data = {
  * Calls SAVEDESTRUCTOR_X if needed, then Simple_vFAIL()
  */
 #define vFAIL(m) STMT_START {                           \
-    PREPARE_TO_DIE;                                     \
     Simple_vFAIL(m);                                    \
 } STMT_END
```

Comment:
This could be simplified to:
```c
#define vFAIL(m) Simple_vFAIL(m)
```

Ditto (mutatis mutandis) for `vFAIL2`, `vFAIL3`, `vFAIL4`, `vFAIL2utf8f`, `vFAIL3utf8f` below. (Alternatively, the `vFAIL` wrappers could be removed entirely.)